pkg/utils/mime.go

```diff
@@ -0,0 +1,118 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
```

Comment:
Should we put this in the bootstrap package since it's specific to bootstrap parsing and probably isn't generalizable to other usage throughout the code?

Reply:
I factored it out for use in the launchtemplate suite test, could definitely move it to the bootstrap package.

Reply:
It would make sense to me as well if this was in the bootstrap package, since we only need it for testing in the launchtemplate suite

Reply:
Ya, I don't see any reason this would ever be used outside the eksboostrap, nodeadm, or launchtemplate test. I'm going to move 👍